### PR TITLE
2017 04 pv refactor item forms

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: apps/contrib/templates/meinberlin_contrib/item_form.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="l-wrapper">
+    <nav class="breadcrumbs">
+        <ul>
+            {% block breadcrumbs %}
+            <li>You need to overwrite the breadcrumbs block</li>
```
Comment (xi) [inline apps/contrib/templates/meinberlin_contrib/item_form.html]: I would prefer to have a meaningful default here, e.g. a link to the homepage.

File: apps/contrib/templates/meinberlin_contrib/item_form.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            {% endblock %}
+        </ul>
+    </nav>
+
+    {% comment %} reuse title as heading {% endcomment %}
+    <h1>{% block title %}You need to overwrite the title block.{% endblock %}</h1>
```
Comment (xi) [inline apps/contrib/templates/meinberlin_contrib/item_form.html]: I would prefer to have a meaningful default here, e.g. "edit".

Comment (vellip) [inline apps/contrib/templates/meinberlin_contrib/item_form.html]: It isn't always edit though, it's either edit or create.

**Comment by vellip** [comment]
I wanted to have one system for all the templates. In my opinion it would've made the template system even more complex if we have extends for some templates but includes for others. Having it like this doesn't introduce more complexity than the refactoring of the detail pages introduces. 
If it's about reducing as much code as possible: Thinking about it, what might still reduce code would be to introduce an item_edit_form and an item_create_form template

**Comment by xi** [comment]
> In my opinion it would've made the template system even more complex if we have extends for some templates but include for others. 

I don't competely agree. We should be very specific about when to use which (and "always use extend" is very specific). But I think there is a reason most templating systems provide both options.

**Comment by xi** [comment]
See #276

**Pull request closed without merging**